### ISSUE TITLE
Mark a media as non seekable

### ISSUE THREAD

**Issue by Tolriq** [androidx/media#591]
It seems in some case ExoPlayer will try to seek when it have buffer underrun due to bad network or low network quality. (I guess but don't know all the cases)

Some sources I have (mainly when transcoding on server side) are not seekable and this cause the media to start again from the start.

Is there a way to tell ExoPlayer to not do that for some media but not the others, or a way to prevent that behaviour completely via a parameter somewhere?

The other solution would be that I could intercept that internal seek and restart the transcoding at the new requested timing (So updating the media url with new parameters basically)

**Comment by Tolriq** [comment]
No I can't this is produced by users private servers during transcoding and with bad network connection.

I can try to gather more logs and details, but I think this is standard default Exoplayer behavior with some kind of media when there's a too large gap it tries to seek to the position it should be no?

**Comment by marcbaechinger** [comment]
Yes, additional logs including `EventLogger` logs would be helpful.

**Comment by Tolriq** [comment]
Unstable. I need to have a nice user contacting me now.
